guide/src/class.md

```diff
@@ -194,15 +197,14 @@ impl SubClass {
 }
 ```
 
-`ObjectProtocol` trait provides `get_base()` method. It returns reference to instance of
-base class.
+The `ObjectProtocol` trait provides a `get_base()` method, which returns a reference
+to the instance of the base struct.
```

Comment:
```suggestion
to an instance of the base struct.
```

Unless I'm misunderstanding this sentence - is there a specific instance this refers back to?

Reply:
As I understand it, it is the specific instance that forms part of the derived object.

Reply:
Ah, withdrawn.